code/modules/mining/equipment/explorer_gear.dm

```diff
@@ -46,3 +46,75 @@
 /obj/item/clothing/mask/gas/explorer/folded/New()
```

Comment:
it's probably unrelated to the main pr but can you change this to initialize() while you're at it?